tests/db_functions/math/test_log.py

```diff
@@ -0,0 +1,34 @@
+import math
+from decimal import Decimal
+
+from django.db.models.functions import Log
+from django.test import TestCase
+
+from ..models import DecimalModel, IntegerModel
+
+
+class LogTests(TestCase):
+
+    def assertLogEqual(self, n_log, n1, n2):
+        diff1 = abs(float(n_log) - math.log(n2, n1))
+        diff2 = abs(float(n_log) - math.log(n1, n2))
+        if(diff1 < diff2):
```

Comment:
Drop parentheses and add a space after `if`: `if diff1 < diff2:`

This feels like it was trying to work around parameters being passed in the wrong order...